MIGRATING_TO_V8.0.0.MD

```diff
@@ -207,3 +207,12 @@ These are the affected properties, listed by the relevant SDK method:
   - Response: `ALL KEYS`
 - `exh.templates.resolveAsJsonUsingCode`
   - Response: `ALL KEYS`
+
+## Unauthenticated requests
+For all [unauthenticated methods](#unauthenticated-methods), in v8.0.0 the response from these requests will now correctly undergo the same transformation steps as any other request.
```

Comment:
BTW the anchors provided in the section above this in this file are wrong. 😞 

###-affected-properties -> #affected-properties. Didn't want to make this change in this PR for clarity. But I can do it before we merge this? 🤔